### PR TITLE
Added Thank You In Sesotho

### PR DESCRIPTION
**Pull request by Sharatsa**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://Sharatsa.github.io/gci15.fossasia.org

**Comment by Sharatsa** [comment]
Added My Geolocation Project

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+﻿---
 ---
```
Comment (mananwason) [inline index.html:2]: @Sharatsa Why have you changed this?

**Comment by mananwason** [comment]
@Sharatsa there is something wrong in your code. Please check once!
screen shot 2016-01-21 at 6 45 20 pm

**Pull request closed without merging**